src/OrbitQt/CaptureOptionsDialog.ui

```diff
@@ -58,6 +58,20 @@
           </property>
          </widget>
         </item>
+        <item>
+         <widget class="QCheckBox" name="apiCheckBox">
+          <property name="text">
+           <string>Enable Orbit Api in target</string>
+          </property>
+         </widget>
+        </item>
+        <item>
+         <widget class="QCheckBox" name="introspectionCheckBox">
+          <property name="text">
+           <string>Enable introspection</string>
+          </property>
+         </widget>
+        </item>
```

Comment:
Basically what Ronald said. I think that we should hide this when not in `--devmode`, and also make sure that the when not in `--devmode` introspection is disabled no matter what the last setting for it was.

Reply:
Yep, done and done, thanks.